flink/v1.20/flink-runtime/NOTICE

```diff
@@ -7,10 +6,48 @@ The Apache Software Foundation (http://www.apache.org/).
 
 --------------------------------------------------------------------------------
 
-This binary artifact includes Apache ORC with the following in its NOTICE file:
+This binary artifact includes Apache Avro 1.12.0 with the following in its NOTICE file:
+
+| Apache Avro
+| Copyright 2010 The Apache Software Foundation
+|
+| This product includes software developed at
+| The Apache Software Foundation (http://www.apache.org/).
+|
+| C JSON parsing provided by Jansson and
+| written by Petri Lehtinen. The original software is
+| available from http://www.digip.org/jansson/.
```

Comment:
Where is this portion coming from? 

https://github.com/apache/avro/blob/main/NOTICE.txt

Reply:
It was coming from Avro 1.7.4. But we actually use 1.12.0. Let me fix.